.travis.yml

```diff
@@ -4,7 +4,7 @@ language: node_js
 
 # https://github.com/nodejs/Release
 node_js:
-  - '11'
+  - '12'
```

Comment:
should we add 14?

Reply:
Sure. Should we remove Node v8 ? :) It gives errors and it's oddly outdated.

Reply:
yep, let's deprecated it